app/Phragile/PhabricatorAPI.php

```diff
@@ -116,4 +116,23 @@ public function queryPHIDs(array $phids)
 			['phids' => $phids]
 		);
 	}
+
+	public function getSprintDuration($phid)
+	{
+		try
+		{
+			$duration = $this->client->callMethodSynchronous(
+				'sprint.getstartenddates',
+				['project' => $phid]
+			);
```

Comment:
This only works if the sprint extension is installed, right? That shouldn't be a problem though as it seems to be an optional dependency only :-)